packages/installation-tests/installation-tests.sh

```diff
@@ -119,9 +116,21 @@ function test_skip_browser_download {
 function test_playwright_should_work {
   initialize_test "${FUNCNAME[0]}"
 
-  npm install ${PLAYWRIGHT_TGZ}
+  OUTPUT=$(npm install ${PLAYWRIGHT_TGZ})
+  if [[ ! "${OUTPUT}" == *"chromium"* ]]; then
```

Comment:
this is brave! NPM outputs all kind of stuff, including ads 😱. I'd prefer not looking into its logs.

If you strongly prefer doing this, I'd do case-insensetive compare at least.

nit: there's "not equal" in bash;

```sh
if [[ "${OUTPUT}" != *"chromium"* ]]; then
```